x-pack/plugins/fleet/common/types/models/agent.ts

```diff
@@ -415,3 +415,12 @@ export interface FleetServerAgentAction {
 
   [k: string]: unknown;
 }
+
+export interface ActionStatusOptions {
+  errorSize: number;
+  page?: number;
+  perPage?: number;
```

Comment:
isn't only `errorSize, page, perPage` used?

Reply:
Yes, I checked again. Probably as you said the other parameters were leftovers!